feconf.py

```diff
@@ -413,10 +416,12 @@ def get_empty_ratings():
 LIBRARY_INDEX_DATA_URL = '/libraryindexhandler'
 LIBRARY_SEARCH_URL = '/search/find'
 LIBRARY_SEARCH_DATA_URL = '/searchhandler/data'
+LOGIN_URL = '/_ah/login'
```

Comment:
We shouldn't expose this particular URL generally; it's only for the perf tests. I'd declare this constant there. (I don't want devs to use it by accident; they should use the existing custom get_login_url() functions.)